src/cuda/compute_add_sources_viscoacoustic_cuda.cu

```diff
@@ -87,34 +88,27 @@ __global__ void compute_add_moving_sources_acoustic_kernel(realw* potential_dot_
                                                     int* ispec_selected_source_moving,
                                                     int* ispec_is_acoustic,
                                                     realw* kappastore,
-                                                    int it, int nsources_local,
-                                                    int max_nsources_local, int NSTEP) {
+                                                    int it, int* nsources_local,
```

Comment:
great to have the GPU version updated as well!

Reply:
It was actually trickier... but yeah GPU is the future!